docs/explanations/introduction.md

```diff
@@ -0,0 +1,29 @@
+# Morpho Introduction
+
+This document explains what Morpho is and what it can be used for.
+
+Fundamentally, Morpho monitors a value on each node in a distributed network, and informs all nodes when they all agree on the same value. The implementation for this uses [OBFT](https://iohk.io/en/research/library/papers/ouroboros-bfta-simple-byzantine-fault-tolerant-consensus-protocol/), which provides resilience against a limited number of byzantine nodes. The network requires a fixed number of nodes, with each node knowing and trusting the public key of every other node in the network.
+
+The main intended use case is to secure proof-of-work (PoW) blockchains against 51% attacks with checkpointing, as described in [this paper](https://iohk.io/en/research/library/papers/securing-proof-of-work-ledgers-via-checkpointing/). More specifically to secure the ETC chain, as proposed in [ECIP1097](https://ecips.ethereumclassic.org/ECIPs/ecip-1097), whose description of a checkpointing federation Morpho is based on. While the implementation has specifics to this use case, there is very little preventing it to be more generic.
+
+## Checkpointing and 51% attacks
+
+51% attacks are possible when an adversary has more than half of the hashing power in the entire network, meaning they can produce blocks at a faster rate than all other miners combined. Over time this allows them to create a forking chain in private which is arbitrarily longer than the longest public chain. At any point the attacker can then disclose their private chain to the public, at which point everybody will switch to the attackers chain, since it's longer. This is a problem because the attacker can choose which transactions are included in their chain. As an example, double spending is possible by not including an already considered confirmed transaction from the public chain in their private chain.
+
+An important aspect to consider is that 51% attacks become easier with lower mining participation, since an attacker only needs more mining power than the entirety of the lawful miners. Because of this, unpopular networks are at an increased risk, which can especially happen when they're new, or towards the end of their lifetime.
```

Comment:
participation, -> participation
unpopular networks -> networks with low mining participation
[It's up to you, but I don't want to admit that our network is unpopular, Sounds like it is not good.]
new, -> new

Reply:
How about this:

> An important aspect to consider is that 51% attacks become easier with lower mining participation since an attacker only needs more mining power than the entirety of the lawful miners. New chains or chains that are being migrated away from are therefore especially at risk.